requirements/constraints.txt

```diff
@@ -14,5 +14,11 @@ Django<5.0
 celery>=5.2.2,<6.0.0
 pylint<3.0
 pycodestyle<2.9.0
-pytz<2023
-testfixtures==7.1.0
+pytz<2024.1
+testfixtures==8.2.0
+
+# Common constraints for Open edX repos
+-c https://raw.githubusercontent.com/openedx/edx-lint/master/edx_lint/files/common_constraints.txt
```

Comment:
@BryanttV, have we checked if the constraints in this plugin are still necessary? I think some of them might not be updated or using the same versions as the main platform as stated [here](https://github.com/eduNEXT/eox-audit-model/blob/3ff92dfeb488922d26d0bd2c5f0fc277ed54cb2c/requirements/constraints.txt#L11). However, to prevent delaying the closure of this PR, I'll open a new issue so we can review and verify the constraints later

Reply:
We might override constraints if we move the file -- we'll have to check though, I'm not 100% sure. If that's the case, this should be done when reviewing the constraints.

Reply:
> have we checked if the constraints in this plugin are still necessary? 

Hi @magajh, the constraints that are right now are aligned with the versions in the Redwood release of edx-platform. But we could check if it is really necessary to keep them in another issue.

Reply:
> Can we please include this constraints at the beginning of this file?
>
> Also, this linked file has the same Django constraint specified in this constraints file, so I believe we can remove the constraint

You can see the changes here: https://github.com/eduNEXT/eox-audit-model/commit/61b5491bfacbd94ccac57536da9f6b2da39e5140

Reply:
@mariajgrimaldi, there are no duplicate constraints in either file
https://raw.githubusercontent.com/openedx/edx-lint/master/edx_lint/files/common_constraints.txt
https://github.com/eduNEXT/eox-audit-model/blob/61b5491bfacbd94ccac57536da9f6b2da39e5140/requirements/constraints.txt